File_handling/fileOperations.test.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+
+const {persons, saveDataToFile} = require('./fileOperations.js')
+
+test('Reading data from file', () => {
+    const readMe = 'persons.json'
+    const testData = fs.readFileSync(readMe, 'utf-8')
+    const expectedData = JSON.parse(testData)
+    expect(persons).toEqual(expectedData)
```

Comment:
It's better to check if the contents equals a certain statically defined contents in the code itself to be clear what are you testing.

Like expect(result).toStrictEqual({mycontent:"value"})